float32 rollRate
float32 pitchRate
float32 yawRate
int32 thrust
